AWS_SETUP.md

```diff
@@ -55,12 +57,13 @@
 
 1. Navigate to your test repository
 2. Visit `Settings` >> `Webhooks`, and click `Add Webhook`
-3. In `Payload` url, enter the `Invoke URL` you copied from the `API Gateway` setup step, and add `/event` to the end of it
-4. On the CLI, run `ruby -rsecurerandom -e 'puts SecureRandom.hex(20)'`. Then add the value to the `Secret` field. **Important**: Save this value somewhere safe
-5. Choose `Let me select individual events.`, and choose `Issues` and `Pull request`
-6. Click `Add webhook`
-7. Navigate to the [`Tokens`](https://github.com/settings/tokens) page in your `GitHub` account settings
-8. Create a new API Token, and give it the `Repo` permissions. Make sure to save this token!
+3. In `Payload URL`, enter the `Invoke URL` you copied from the `API Gateway` setup step, and add `/event` to the end of it
+4. In `Content type`, select `application/json`
```

Comment:
We could also write the list like:
```
1. List item 1
1. List item 2
1. List item 3
1. etc.
```
That would render to:
1. List item 1
1. List item 2
1. List item 3
1. etc.

With this style, I don't have to update the whole list to make it look neat 😛